OpenQuestion/answer-question-6.md

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
Duplicate file below, please help to recheck

Reply:
Remove redundant empty line

Reply:
I fixed it ,thanks